less/plugins/adapt-contrib-slider/slider.less

```diff
@@ -21,9 +21,12 @@
   }
 
   &__number-selection {
```

Comment:
Suggest removing the additional styles and instead change the variables colours alongside the introduction of a box-shadow style that does not impact the elements height.

```
&__number-selection {
  background-color: @font-color-inverted;
  color: @font-color;
  z-index: 5;
}

&:not(.is-complete) &__number-selection {
  box-shadow: 0 0 0 2px @item-color inset;
}
```